tasks/copy_from_archive/requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto3==1.18.40
 fastjsonschema==2.15.0
 aws_lambda_powertools==1.31.0
-../../shared_libraries[recovery]
\ No newline at end of file
+../../shared_libraries[recovery]
+psycopg2-binary==2.8.6
```

Comment:
The latest version is 2.9.9. Do we still want to use 2.8.6?

Reply:
Think that's a good idea since 2.8.6 is going on 4 years old. Updated the requirements-dev.txt as well